<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kostki</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="diceNoSpin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="rollNoSpin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba kostek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba rzutów</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Żądana kombinacja</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="combEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="resultLCD">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ExecButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetlać pasujące kombinacje?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Szansa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pasujące kombinacje</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="faceNoSpin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba ścian</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
